external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -2461,6 +2491,10 @@ impl<'a> ParsingSymbolicator<'a> {
                         }
                     };
 
+                    fun.attributes.iter().for_each(|sp!(_, attrs)| {
+                        attrs.iter().for_each(|a| self.attr_symbols(a.clone()))
```

Comment:
Is there a reason we need the clone here? Can we rewrite `attr_symbols` to work over `&P::Attribute` and save ourselves some grief?